src/t_zset.c

```diff
@@ -1790,17 +1792,20 @@ void zaddGenericCommand(client *c, int flags) {
     scores = zmalloc(sizeof(double) * elements);
     for (j = 0; j < elements; j++) {
         if (getDoubleFromObjectOrReply(c, c->argv[scoreidx + j * 2], &scores[j], NULL) != C_OK) goto cleanup;
+        ele = c->argv[scoreidx + 1 + j * 2]->ptr;
+        size_t elelen = sdslen(ele);
+        if (maxelelen < elelen) maxelelen = elelen;
```

Comment:
I think this is more intuitive most of the time.
```suggestion
        if (elelen > maxelelen) maxelelen = elelen;
```